netdata.spec.in

```diff
@@ -150,17 +169,16 @@ rm -rf $RPM_BUILD_ROOT
 %defattr(-,root,root)
 
 %dir %{_sysconfdir}/%{name}
+%dir %{_sysconfdir}/%{name}/health.d
+%dir %{_sysconfdir}/%{name}/python.d
+
 %config(noreplace) %{_sysconfdir}/%{name}/*.conf
 #%config(noreplace) %{_sysconfdir}/%{name}/charts.d/*.conf
 %config(noreplace) %{_sysconfdir}/%{name}/health.d/*.conf
 #%config(noreplace) %{_sysconfdir}/%{name}/node.d/*.conf
 %config(noreplace) %{_sysconfdir}/%{name}/python.d/*.conf
 %config(noreplace) %{_sysconfdir}/logrotate.d/%{name}
 
-# node.js config files are JSON (which do not support comments)
-# so, we only supply markdown files
-%{_sysconfdir}/%{name}/node.d/*.md
-
```

Comment:
Package was failing build on all platforms (Fedora/CentOS/OpenSUSE), as no `/etc/netdata/node.d` directory was ever found in the buildroot.

@ktsaou can you confirm this?